pkg/costmodel/costmodel.go

```diff
@@ -2555,8 +2555,7 @@ func (cm *CostModel) QueryAllocation(window kubecost.Window, resolution, step ti
 
 					if totals == nil {
 						log.Errorf("unable to locate asset totals for allocation %s", key)
-						return nil, fmt.Errorf("unable to locate allocation totals for allocation")
-
+						continue
```

Comment:
I think this is good @r2k1 . For others: this changes the behavior of a PARC for which we cannot locate totals for to simply not be included, rather than bubbling up an error. 

This doesn't fix whatever the underlying issue is with no totals in the store, but this at least lets us degrade gracefully